apps/language_server/lib/language_server/packet_router.ex

```diff
@@ -0,0 +1,92 @@
+defmodule ElixirLS.LanguageServer.PacketRouter do
```

Comment:
Why not use `Registry` or `:pg` instead?

Reply:
Mainly because this is intended to be a temporary module that will get removed and it's only doing its work for two processes.